frameworks/kafka/README.md

```diff
@@ -1,14 +1,420 @@
-# DC/OS Apache Kafka Service Documentation
+<p align="left"><img src="https://mesosphere.com/wp-content/themes/mesosphere/library/images/assets/dcos-sdk-logo.png" width="250"/></p>
+
+
+# Apache Kafka Service Guide
 
 ## Table of Contents
 
-- [Overview](docs/index.md)
-- [Quick Start](docs/quick-start.md)
-- [Install and Customize](docs/install-and-customize.md)
-- [Uninstall](docs/uninstall.md)
-- [Configure](docs/configure.md)
-- [Connecting Clients](docs/connecting-clients.md)
-- [Managing](docs/managing.md)
-- [API Reference](docs/api-reference.md)
-- [Troubleshooting](docs/troubleshooting.md)
-- [Limitations](docs/limitations.md)
+- [Overview](#overview)
+  - Features
+- [Quick Start](#quick-start)
+- [Installing and Customizing](#installing-and-customizing)
+  - Installation with Default Settings
+  - Installation with Custom Settings
+  - Service Settings
+    - Service Name
+    - _SERVICE-WIDE OPTIONS SPECIFIC TO YOUR IMPLEMENTATION GO HERE_
+  - Node Settings
+    - Node Count
+    - CPU
+    - Memory
+    - Ports
+    - Storage Volumes
+    - Placement Constraints
+    - _PER-NODE OPTIONS SPECIFIC TO YOUR IMPLEMENTATION GO HERE_
+  - _STEP-BY-STEP GUIDES FOR ANY ADDITIONAL CONFIG SCENARIOS TO POINT OUT_
+- [Uninstalling](#uninstalling)
+- [Connecting Clients](#connecting-clients)
+  - Discovering endpoints
+  - Connecting clients to endpoints
+- [Managing](#managing)
+  - Updating Configuration
+    - Adding a Node
+    - Resizing a Node
+    - _PER-NODE OPTIONS SPECIFIC TO YOUR IMPLEMENTATION GO HERE_
+    - Updating Placement Constraints
+    - _SERVICE-WIDE OPTIONS SPECIFIC TO YOUR IMPLEMENTATION GO HERE_
+  - Restarting nodes
+  - Replacing nodes
+- [Disaster Recovery](#disaster-recovery)
+  - Backup
+  - Restore
+- [Deployment Best Practices](#deploy-best-practices)
+- [Troubleshooting](#troubleshooting)
+  - Accessing Logs
+- [Known Issues](#knownissues)
+- [Limitations](#limitations)
+  - Removing a Node
+  - Automatic Failed Node Recovery
+  - Updating Storage Volumes
+  - Rack-aware Replication
+  - _ANY OTHER CAVEATS TO MENTION HERE?_
+- [Terms](#terms)
+- [Support](#support)
+  - Supported Versions
+  - Package Versioning
+    - Upgrades/downgrades
+  - Reaching Technical Support
+
+<a name="overview"></a>
+# Overview
+
+DC/OS Apache Kafka is an automated service that makes it easy to deploy and manage Apache Kafka on [Mesosphere DC/OS](http://dcos.io). Apache Kafka is a distributed high-throughput publish-subscribe messaging system with strong ordering guarantees. Kafka clusters are highly available, fault tolerant, and very durable. For more information on Apache Kafka, see its [documentation](http://kafka.apache.org/documentation.html).
```

Comment:
Yes, and the link should got to our enterprise site, not the OSS site. I will fix.